LinuxPackaging.cmake

```diff
@@ -0,0 +1,95 @@
+# support creating some basic binpkgs via `make package`
+set(CPACK_GENERATOR TGZ;RPM;DEB)
+
+set(CPACK_INCLUDE_TOPLEVEL_DIRECTORY 0)
+set(CPACK_PACKAGE_VERSION_MAJOR ${LIBIIO_VERSION_MAJOR})
+set(CPACK_PACKAGE_VERSION_MINOR ${LIBIIO_VERSION_MINOR})
+set(CPACK_PACKAGE_VERSION_PATCH g${LIBIIO_VERSION_GIT})
+set(CPACK_BUNDLE_NAME libiio)
+set(CPACK_PACKAGE_VERSION ${LIBIIO_VERSION})
+# debian specific package settings
+set(CPACK_PACKAGE_CONTACT
+       "Engineerzone <https://ez.analog.com/community/linux-device-drivers>")
+
+FIND_PROGRAM(DPKG_CMD dpkg)
+FIND_PROGRAM(DPKGQ_CMD dpkg-query)
```

Comment:
You could do:
if (NOT CPACK_DETECT_DEPENDENCIES)
return()
endif()

That would avoid to look up for dpkg and dpkg-query if CPACK_DETECT_DEPENDENCIES is not set.